community/planning/features.md

```diff
@@ -22,3 +22,10 @@ and services.
 
 We welcome discussion on these topics, either on Slack or during the engineering
 forum.
+
+## Features
+
+* [Canopy 0.2]({% link community/planning/canopy_0.2.md %})
```

Comment:
We just use direct links elsewhere in the docs (`/community/planning/canopy_0.2.html`). We should probably decide which format we're going to use and stick with it.

Reply:
These are actually checked at build time, so if a file is moved and a link is not updated, the site builder will return an error.

Reply:
Sweet, that seems like a compelling reason to do things this way.